constrained-voucher.mkd

```diff
@@ -627,8 +628,7 @@ the voucher first the abstract (tree) definition as explained
 in {{-yang-tree}}.  This provides a high-level
 view of the contents of each artifact.
 
-Then the assigned SID values are presented. These have been assigned using
-the rules in {{I-D.ietf-core-sid}}.
+The yang voucher and voucher-request are encoded following the cbor rules defined {{RFC7049}}. An JSON array is encoded as a a CBOR array, and a JSON item is encoded as a CBOR map. In addition the cbor json identifiers that serve as map keys are converted to numbers, called Voucher Keys (VK) as also proposed in {{I-D.ietf-core-sid}}.
```

Comment:
I don't understand the mention of JSON here.
We don't need to remove the references, we just need to soften them so that they can be informative.